xbmc/media/MediaType.cpp

```diff
@@ -75,6 +75,14 @@ MediaType MediaTypes::ToPlural(const MediaType &mediaType)
   return mediaTypeIt->second.plural;
 }
 
+bool MediaTypes::IsCollection(const MediaType &mediaType)
```

Comment:
It's not needed at all. `MediaTypes::IsContainer()` is what we want and it already exists.